src/test/java/org/zalando/planb/revocation/api/RevocationResourceTest.java

```diff
@@ -181,6 +182,26 @@ public void testBadRequestWhenPostingFutureRevocation() throws Exception {
     }
 
     /**
+     * Tests that when {@code POST}ing revocations with an ancient {@code issued_before} field, a HTTP {@code BAD_REQUEST}
+     * is returned.
+     *
+     * <p>Furthermore asserts that a standard {@link Problem} is returned.</p>
+     */
+    @Test
+    public void testBadRequestWhenPostingAncientRevocation() throws Exception {
```

Comment:
If maxTimeDelta > ONE_YEAR_IN_SECONDS this fails. So this must be relative to it.